backend/src/controllers/UserController.js

```diff
@@ -0,0 +1,19 @@
+import User from '../models/User';
+import { UserService } from '../services/UserService';
+
+export class UserController {
+  constructor() {
+    this.userService = new UserService();
+  }
+
+  async register(req, res) {
+    const user = new User(req.body.name, req.body.email, req.body.password);
+    console.log(user);
+    return await this.userService
+      .registerUser(user)
+      .then(data => res.status(201).json(data))
+      .catch(err => {
+        res.status(400).json(err.message);
+      });
```

Comment:
@marcianszlovak either use async / await or then callbacks but don't mix it
```suggestion
  try {
    let data = await this.userService.registerUser(user);
    res.status(201).json(data);
  } catch(err) {
    res.status(400).json(err.message);
  }
```